mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksView.kt

```diff
@@ -295,4 +300,31 @@ interface MavericksView : LifecycleOwner {
         onFail: (suspend (Throwable) -> Unit)? = null,
         onSuccess: (suspend (T) -> Unit)? = null
     ) = _internalSF(subscriptionLifecycleOwner, asyncProp, deliveryMode, onFail, onSuccess)
+
+    /**
+     * Subscribes to on the given Flow. This can be utilized to create customized subscriptions that for example drop the first
+     * element in the flow before continuing.
+     *
+     * @param deliveryMode If [UniqueOnly], when this [MavericksView] goes from a stopped to start lifecycle a value
+     * will only be emitted if the value has changed. This is useful for transient views that should only
+     * be shown once (toasts, poptarts), or logging. Most other views should use false, as when a view is destroyed
+     * and recreated the previous state is necessary to recreate the view.
+     *
+     * Use [uniqueOnly] to automatically create a [UniqueOnly] mode with a unique id for this view.
+     *
+     * @param action supports cooperative cancellation. The previous action will be cancelled if it as not completed before
+     * the next one is emitted.
+     */
+    fun <T : Any> Flow<T>.collectLatest(
+        deliveryMode: DeliveryMode,
```

Comment:
use a default value of RedeliverOnStart here, since that is what most people need?

Reply:
oh, maybe requiring that is needed to disambiguate it from the normal collectLatest

Reply:
hmm it should be fine, the regular one doesnt have defaults.